### PR TITLE
[bitnami/ejbca] Provides ingress access 

### PR DESCRIPTION
**Pull request by rafariossaa**
**Description of the change**

Provides ingress access to EJBCA

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [-] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by rafariossaa** [comment]
This needs an updated version of the EJBCA container that provides the new environment variable.

**Comment by carrodher** [comment]
You can attach this PR to our internal CI/CD pipeline and the PR will be updated with the latest tags and version

File: bitnami/ejbca/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ejbca
-version: 0.2.1
+version: 0.2.2
```
Comment (carrodher) [inline bitnami/ejbca/Chart.yaml:3]: As it is a new feature adds more than bug fixes I would bump the chart version in a minor:
```suggestion
version: 0.3.0
```

File: bitnami/ejbca/values.yaml [hunk 2]
```diff
@@ -215,7 +215,13 @@ service:
   port: 8080
   ## HTTPS Port
   ##
+  # httpsPort: 8443
   httpsPort: 8443
+  ## HTTPS Advertised port
+  ##
+  #
```
Comment (carrodher) [inline bitnami/ejbca/values.yaml:222]: This is not needed

**Review by carrodher — APPROVED** [review]

**Pull request closed without merging**